### PR TITLE
Resize parameter in the files download endpoint

### PR DESCRIPTION
**Pull request by iamdevice**

### CODE REVIEW

File: openapi/components/parameters/resize.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+name: resize
+in: query
+description: Resize image to specified size. Support any sizes from 10x10 to 2000x2000.
```
Comment (ashkarpetin) [inline openapi/components/parameters/resize.yaml:3]:
```suggestion
description: Resize image to specified size. Supports any sizes from 10x10 to 2000x2000.
```

**Review by subenson — DISMISSED** [review]

**Review by subenson — DISMISSED** [review]

**Review by ashkarpetin — DISMISSED** [review]

File: openapi/components/parameters/resize.yaml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+name: resize
```
Comment (slavcodev) [inline openapi/components/parameters/resize.yaml:1]:
```suggestion
name: size
```

?

Comment (slavcodev) [reply openapi/components/parameters/resize.yaml:1]:
```suggestion
name: imageSize
```
?

File: openapi/components/parameters/resize.yaml [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+in: query
+description: Resize image to specified size. Supports any sizes from 10x10 to 2000x2000.
```
Comment (slavcodev) [inline openapi/components/parameters/resize.yaml:3]: So the format is `{width}x{height}` ? or `{height}x{width}` ? What happens for invalid value?

Comment (iamdevice) [reply openapi/components/parameters/resize.yaml:3]: `{width}x{height}`. If received invalid value we return an original-sized image.

Comment (slavcodev) [reply openapi/components/parameters/resize.yaml:3]: This info would be fine to add in the description. Or may be consider to use two difefrent params `imageWidth` and `imageHeight`

File: openapi/components/parameters/imageResize.yaml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+name: imageResize
```
Comment (slavcodev) [inline openapi/components/parameters/imageResize.yaml:1]: I believe `resize` is not quite correct word to use in param name. 

```suggestion
name: imageSize
```

Comment (iamdevice) [reply openapi/components/parameters/imageResize.yaml:1]: We resize an original image

File: openapi/components/parameters/imageSize.yaml [hunk 5]
```diff
@@ -0,0 +1,8 @@
+name: imageSize
+in: query
+description: >-
+  Resize image to specified size. Supports any sizes from 10x10 to 2000x2000 (format `{width}x{height}`).
+  The image will be returned in the original size if the value is invalid. This parameter will be ignored for non-image files.
+schema:
+  type: string
+  example: 700x700
```
Comment (slavcodev) [inline openapi/components/parameters/imageSize.yaml:8]:
```suggestion
  example: 700x700
  pattern: \d+x\d+
```

**Review by slavcodev — APPROVED** [review]

**Review by ashkarpetin — APPROVED** [review]

**Pull request merged**